www/templates/html/scripts/frontend.js

```diff
@@ -0,0 +1,13 @@
+require(['jquery'], function($){
+	WDN.initializePlugin('modal', [function() {
```

Comment:
I think you can use require to initialize this plugin too. That might be more consistent.